### DISCUSSION TITLE
How do I activate a gameobject?

### DISCUSSION THREAD

**Discussion by aarnithesuperproguyprosuperguypropro** [vfsfitvnm/frida-il2cpp-bridge#375] [Q&A]
I'm trying to activate a game object that is disabled in the game by default.

**Comment by aarnithesuperproguyprosuperguypropro** [comment]
Some extra info: There's also a function that will enable the gameobject on startup if a bool is set to true

**Reply by 350030173** [reply]
find the gameobject and invoke this method?
// Namespace: UnityEngine
public sealed class GameObject : Object
public void SetActive(bool value);

**Reply by aarnithesuperproguyprosuperguypropro** [reply]
how would I find the game object?
